region/brinstar/red/Beta Power Bomb Room.json

```
@@ -137,6 +137,55 @@
         "If any Sidehoppers remain after using a Power Bomb, safely kill them from the hole below."
       ]
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway (Morph Ball Dodge)",
+      "requires": [
+        {"obstaclesNotCleared": ["B"]},
+        "Morph"
```

This will put it in Hard in Map Rando. Could also just be a careful jump, but its kind of not obvious at lower difficulties - the hoppers can be scary
```suggestion
        "Morph",
        "canDodgeWhileShooting"
```